templates/cassiopeia/html/layouts/chromes/card.php

```diff
@@ -11,23 +11,31 @@
 
 use Joomla\Utilities\ArrayHelper;
 
-$module  = $displayData['module'];
-$params  = $displayData['params'];
-$attribs = $displayData['attribs'];
+$module		= $displayData['module'];
+$params		= $displayData['params'];
+$attribs	= $displayData['attribs'];
```

Comment:
```suggestion
$module  = $displayData['module'];
$params  = $displayData['params'];
$attribs = $displayData['attribs'];
```
Please revert your change here.

Reply:
Reverted back